monkey/tests/data_for_tests/monkey_configs/flat_config.json

```diff
@@ -66,7 +66,6 @@
         "SMBFinger",
         "SSHFinger",
         "HTTPFinger",
-        "MySQLFinger",
```

Comment:
Why remove this in this PR?

Reply:
Because we know it's going away and I'm writing tests that use this data. It'll be one less thing to change when we do #1648. Otherwise, I can add the appropriate asserts in the tests, but then they have to be removed too. In short, to avoid adding code/tests that I know will have to be removed in the coming days.